docs/chart_repository.md

```diff
@@ -271,7 +276,7 @@ If the charts are backed by HTTP basic authentication, you can also supply the
 username and password here:
 
 ```console
-$ helm repo add fantastic-charts https://username:password@fantastic-charts.storage.googleapis.com
+$ helm repo add fantastic-charts https://username:password@fantastic-charts.storage.googleapis.com my-username my-password
```

Comment:
This should probably be updated to `$ helm repo add fantastic-charts https://fantastic-charts.storage.googleapis.com --username my-username --password my-password` once the args are turned into flags.